Sources/StreamChatUI/ChatMessageList/RETHINK/MessageListVC.swift

```diff
@@ -50,7 +106,7 @@ class MessageListVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionVie
         let message = channelController.messages[indexPath.item]
         
         let reuseId = cellReuseIdentifier(for: message)
```

Comment:
Should we also change this to the index path version?

Reply:
Most likely but decided not to touch the reuse logic part in this PR